csharp/ql/lib/semmle/code/csharp/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -1208,6 +1194,11 @@ private module Cached {
     } or
     TCapturedVariableContent(VariableCapture::CapturedVariable v)
 
+  cached
+  newtype TContentSet =
+    TSingletonContent(Content c) { not c instanceof PropertyContent } or
+    TPropertyContentSet(Property p) { p.isUnboundDeclaration() }
```

Comment:
Shouldn't the corresponding `TPropertyContent` injector be deleted from `TContent` if `Content` has been replaced with `ContentSet` everywhere (it looks like it will require a re-write of `ContentApprox`)?

Reply:
We still need `Content`, so that can't be deleted. `ContentSet` is a type used to represent sets of `Content`s with just a single entity.

Reply:
Yes, I get that we need the `Content` type. I am just trying to understand whether we need an injector for properties both as a part of `Content` and `ContentSet`?

Reply:
Ok - thank you!